README.md

```diff
@@ -229,7 +229,7 @@ To run tests locally, run:
 $ npm test
 ```
 
-The test run will also include a run of [eslint](https://eslint.org/). To run the tests without these, use:
+The test run will also include a run of [eslint](https://eslint.org/) and [TypeScript](https://www.typescriptlang.org/). To run the tests without these, use:
 
 ```shell
 $ npm run test-no-lint
```

Comment:
If I'm being picky, this could be changed to `test-no-lint-or-tsc` or `test-no-lint-tsc` or `only-test`